### ISSUE TITLE
FIXME: Database on main crashes a couple of times, process interrupted etc.

### ISSUE THREAD

**Issue by Chunnyluny** [RimSort/RimSort#313]
**Describe the bug**
_A clear and concise description of what the bug is._
Database on main crashes a couple of times, process interrupted etc. What I suspect what I did was, I saw it being quite fast, but it was just validating the mods. Because I have already done 3349 mods inactive it says missing data from some mods, but I believe it tries to use my mods' config, which is probably stupid. It tries to find the mods that are in the Mod_real folder. But anyway, I thought while typing it I should disable validating mods, which I did and then RimSort crashed multiple times. After that, I started the RimSort again, but without validate downloaded mods off and hope for the best. Will post more of my experience after my 3rd or 4th attempt at building a database, because I want to help with updating it using custom steps to follow to get it to work.

Also, I tried to refresh the process when database building was being stuck, and I thought it will wake up the process again.

**To Reproduce**
_Steps to reproduce the behavior:_
1. Go to '...' Settings to start your database building using Steam CMD
2. Click on '....' while in the process see that it takes forever and notice that you forgot to disable validate downloaded mods (will take forever otherwise)

**Expected behavior**
_A clear and concise description of what you expected to happen._
I hoped to not crash the application multiple times.

**Screenshots**
_If applicable, add screenshots to help explain your problem._

![image](https://github.com/RimSort/RimSort/assets/102756485/9c92c337-a48e-482e-8a6e-d714c135dc57)

**Please complete the following information:**
- OS: [e.g., iOS]: Windows 11
- RimSort version: python interpreter, latest main branch
- Please provide a `RimSort.log`:
http://0x0.st/XH5O.old.log
http://0x0.st/XH5V.log
- Are you running from a Python interpreter or from a compiled RimSort release? Python interpreter, I'm testing database building using SteamCMD, with custom steps: rename Mods --> Mods_real make a folder called Mods_dbbuilder. Screens are added to add more clarification to the subject we are talking about. 

**Additional context**
Add any other context about the issue here.

![image](https://github.com/RimSort/RimSort/assets/102756485/b6587a26-7260-4a85-be76-43fdb08f7506)

<details>
Traceback (most recent call last):
File "C:\Users\chunl\Documents\GitHub\RimSort-main\app\windows\runner_panel.py", line 238, in handle_output
self.message(stdout)
File "C:\Users\chunl\Documents\GitHub\RimSort-main\app\windows\runner_panel.py", line 272, in message
self.steamcmd_download_tracking.remove(pfid)
ValueError: list.remove(x): x not in list
</details>

**Comment by twstagg** [comment]
@Chunnyluny this should be fixed now
